### ISSUE TITLE
Empty `staticMembers`

### ISSUE THREAD

**Issue by eernstg** [dart-lang/reflectable#36]
We got reports about `staticMembers` returning the empty list in a case where there are actually several static members; apparently, the expected mirrors are available if an `instanceInvokeCapability` is included, which would be a bug because that area is covered by various kinds of `StaticInvokeCapability`. Details are currently being exchanged via email.

**Comment by eernstg** [comment]
Awaiting further information about the situation that creates the issue.

**Comment by sigurdm** [comment]
This program shows the issue:

```
import 'package:reflectable/reflectable.dart';

class MyReflectable extends Reflectable {
  const MyReflectable() : super(staticInvokeCapability);
}

const myReflectable = const MyReflectable();

@myReflectable
class A {
  static const List<String> foo = const ["apple"];
}

main() {
  InstanceMirror instanceMirror = myReflectable.reflect(new A());
  print(instanceMirror.type.staticMembers);
}
```

Transformed it prints:
```
{}
```

Untransformed it prints:
```
{foo: _MethodMirrorImpl('Instance of '_SyntheticAccessor'')}
```

**Comment by eernstg** [comment]
Thx, this immediately revealed a bug, fixed in https://github.com/dart-lang/reflectable/issues/36.

**Comment by eernstg** [comment]
Landed in a4686098f8db6aa4dc9cfe9e1d452ba2e01ccd4a.
